src/protocol.h

```diff
@@ -351,44 +380,44 @@ public slots:
     void ChangeChanInfo ( CChannelCoreInfo ChanInfo );
     void ReqChanInfo();
     void ChatTextReceived ( QString strChatText );
-    void NetTranspPropsReceived ( CNetworkTransportProps NetworkTransportProps );
+    void
```

Comment:
Ugh.  Split after the `(` after the method name, not after the return type...

Reply:
stays one line with `ColumnLimit: 120`